### DISCUSSION TITLE
Cloudflare worker proxy as a subdirectory problem

### DISCUSSION THREAD

**Discussion by fdocr** [plausible/analytics#1837] [Anything Else]
Hello Plausible team 🙂 

I have a similar problem as https://github.com/plausible/analytics/discussions/1606 where `/api/event` is returning a 404, but my visits aren't being tracked anymore (they no longer appear on `plausible.io` dashboard).

I followed [this Cloudflare guide](https://plausible.io/docs/proxy/guides/cloudflare) and have a worker fully configured with all optional steps. When I visit my `script.js` it does work (I can see the JS code on my browser) but the `/api/event` requests fail with a 404 on the browser:

<img width="1381" alt="Screen Shot 2022-04-16 at 15 08 03" src="https://user-images.githubusercontent.com/6045239/163691576-7a8a3c0d-7539-4027-b38e-9bbdb86e85ae.png">

The weird thing is that I started debugging using Cloudflare dashboard (worker live logs) and requests seem to work as expected inside the worker itself:

<img width="770" alt="Screen Shot 2022-04-16 at 15 09 37" src="https://user-images.githubusercontent.com/6045239/163691592-24c70545-daab-466d-aa04-3008fa9f6532.png">

<img width="794" alt="Screen Shot 2022-04-16 at 15 09 49" src="https://user-images.githubusercontent.com/6045239/163691599-b83a1fce-4bed-42dc-bdf1-afe4ab2bf0b2.png">

Visits used to be tracked before I enabled the proxy (default `plausible.io` script url) but now they aren't anymore. Am I missing something?

**Comment by RobertJoonas** [comment]
Hi @fdocr!

I suspect the custom folder you have specified in the code for the `/api/event` endpoint does not match your actual subfolder. For reference, I'm speaking of the second line of code in [this section](https://plausible.io/docs/proxy/guides/cloudflare#step-3-quick-edit-the-worker). Looking at your screenshots, the value for the second line should be:

```javascript
const Endpoint = '/dolan/wat/event'
```

Let me know if that's the case please, thanks!

**Reply by fdocr** [reply]
Oh yes, it was kind of a typo of mine on that exact line. When following the guide I had a different replacement for `api`, something other than `wat` (so it's also different than the script path) that I didn't match correctly on the script data-tag, where I had copy/pasted `data-api="/dolan/wat/event"`.

Fixing this solved my problem. Thanks a lot for the help @RobertJoonas, everything is working as expected now with the Cloudflare proxy 🙌
